search-service/src/main/kotlin/db/migration/V0_28__JsonLd_migration.kt

```diff
@@ -94,10 +95,15 @@ class V0_28__JsonLd_migration : BaseJavaMigration() {
 
             // in current implementation, geoproperties do not have a creation date as they are stored
             // as a property of the entity node, so we give them the creation date of the entity
-            val defaultCreatedAt = getPropertyValueFromMapAsDateTime(
-                expandedEntity as Map<String, List<Any>>,
-                NGSILD_CREATED_AT_PROPERTY
-            ) ?: ZonedDateTime.parse("1970-01-01T00:00:00Z")
+            val defaultCreatedAt =
+                try {
+                    getPropertyValueFromMapAsDateTime(
+                        expandedEntity as Map<String, List<Any>>,
+                        NGSILD_CREATED_AT_PROPERTY
+                    ) ?: ZonedDateTime.parse("1970-01-01T00:00:00Z")
+                } catch (e: DateTimeParseException) {
+                    ZonedDateTime.parse("1970-01-01T00:00:00Z")
```

Comment:
also add a WARN log (we need to have a trace of what has gone wrong during the migration)